<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryDialog</class>
 <widget class="QDialog" name="HistoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Screenshot History</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../lightscreen.qrc">
    <normaloff>:/icons/view-history</normaloff>:/icons/view-history</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <item>
    <widget class="QLineEdit" name="filterEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="whatsThis">
      <string>Type here to filter through the screenshots in the list.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: palette(mid);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="whatsThis">
      <string>Double click the path to open the image with the default picture viewer, you can also double-click the URL to open it on your web browser.

Right click items to get access to more options.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="uploadButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../lightscreen.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>HistoryDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
